packages/collector/test/tracing/misc/node-fetch/app.js

```diff
@@ -39,8 +39,8 @@ app.get('/', async (req, res) => {
 });
 
 app.get('/request', async (req, res) => {
+  log('Received request /request');
```

Comment:
This log required?

Reply:
Yes it gives visibility while running the tests.

Reply:
I added this in the wrong app. Thanks

Reply:
Reverting.

Reply:
Reverted